### PR TITLE
post about nbdime

### PR DESCRIPTION
**Pull request by minrk**
cc @defeo

### CODE REVIEW

File: _posts/2017-01-13-nbdime.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+layout: post
+title: nbdime released
+author: Min Ragan-Kelley
+type: blogpost
+date: 2017/1/13
+---
+
+[nbdime](https://nbdime.readthedocs.io) has had its first stable release.
+nbdime provides tools for diffing and merging Jupyter notebooks,
+and inegrating notebooks into git workflow.
```
Comment (olexandr-konovalov) [inline _posts/2017-01-13-nbdime.md]: typo 'inegrating'

File: _posts/2017-01-13-nbdime.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- ensuring that merged notebooks are always valid
+- integration with git as `drivers` for passive diff/merge integration and `tools` for interactive GUI integration
+
+Tools provided by nbdime:
+
+- `nbshow`: show a legible formatting of a notebook on the command-line
+- `nbdiff`: command-line diff of notebooks, eliding outputs that are known to not be renderable in a terminal. `nbdiff` can be integrated into git as a diff driver.
+- `nbdiff-web`: create a rich, rendered web view of the changes between two notebooks
+- `nbmerge`: three-way merge with automatic conflict resolution, which can be integrated into git as a merge driver, ensuring always-valid notebooks and eliminating merge conflicts on transient fields.
+- `nbmerge-web`: interactive three-way merge tool for manually resolving merge conflicts
```
Comment (olexandr-konovalov) [inline _posts/2017-01-13-nbdime.md]: I'd like to unify formatting style - some sentences end with `.`, some not...

**Review by olexandr-konovalov — COMMENTED** [review]
The content looks good to me. @defeo could you look at this too, please?

**Comment by olexandr-konovalov** [comment]
Ping @minrk @defeo - looks ok, minimal polishing needed. I could fix and merge.

**Comment by minrk** [comment]
@alex-konovalov sorry about that, I missed the review. Should be fixed now.

**Pull request merged**